ui/src/hosts/containers/HostPage.js

```diff
@@ -150,7 +150,9 @@ export const HostPage = React.createClass({
   },
 
   render() {
-    const {params: {hostID}, location: {query: {app}}, source: {id}, autoRefresh, handleChooseAutoRefresh, inPresentationMode, handleClickPresentationButton} = this.props
+    const {params: {hostID}, location: {query: {app}}, source: {id},
+    autoRefresh, handleChooseAutoRefresh, inPresentationMode,
+    handleClickPresentationButton, source} = this.props
```

Comment:
Otherwise, very good stuff, and good job on the no-user-select mixing, @alexpaxton .